src/courseware/course/content-tools/calculator/Calculator.jsx

```diff
@@ -341,6 +341,7 @@ class Calculator extends Component {
                         <li>arccsch(4x+y)</li>
                       </ul>
                     </td>
+                    {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
                     <td dir="auto" />
```

Comment:
[question]: Can we add some kind of associative label instead of disabling the rule?

Reply:
Added